lib/db/schema/patch-091-092.sql

```diff
@@ -0,0 +1,156 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('91');
+
+CREATE TABLE IF NOT EXISTS `clientsInstances` (
+  `uid` BINARY(16) NOT NULL,
+  `id` BINARY(16) NOT NULL,
+  `clientId` BINARY(8) NOT NULL,
+  `name` VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin,
+  `pushEndpoint` VARCHAR(255),
+  `pushPublicKey` CHAR(88),
+  `pushAuthKey` CHAR(24),
+  PRIMARY KEY (`uid`,`id`),
+  UNIQUE KEY `UQ_uid_id` (`uid`,`id`)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS `clientsInstancesCommands` (
```

Comment:
@eoger I recall a previous conversation where you suggested trying to re-use the existing "deviceCommands" table rather than create a new one, I'm curious why you decided against that route?